src/css_parser.rs

```diff
@@ -406,6 +408,13 @@ pub enum CssBorderParseError<'a> {
     ColorParseError(CssColorParseError<'a>),
 }
 
+impl_display!{ CssBorderParseError<'a>, {
+    InvalidBorderStyle(e) => format!("Invalid border style: {}", e.0),
```

Comment:
Well, I'd keep it consistent with the rest of the formatting - either always use `\"{}\"` or `{}` - I'd prefer the former.